nid_validator/usa_validator/tests/test_views.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from django.test import RequestFactory
+from usa_validator.views import USANIDValidationViewSet
+
+class TestUSANIDValidationViewSet(unittest.TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.view = USANIDValidationViewSet.as_view({'post': 'create'})
+
+    def test_create(self):
```

Comment:
Sweep: Add test case for Delete/GET/PUT methods and make sure it returns the method not allowed.